modules/reindex/src/test/java/org/elasticsearch/reindex/AsyncBulkByScrollActionTests.java

```diff
@@ -498,6 +551,9 @@ public boolean isCancelled() {
             }
         });
 
+        // Set the base for the scroll to wait - this is added to the figure we calculate below
+        firstSearchRequest.scroll(timeValueSeconds(10));
```

Comment:
Could we perhaps remove the field and just use `testRequest.getSearchRequest()` instead? Seems unnecessary to have that field and using it is now slightly trickier due to the copy done.